views/transactions.php

```diff
@@ -1,55 +1,66 @@
 <!DOCTYPE html>
-<html>
-    <head>
-        <title>Transactions</title>
-        <style>
-            table {
-                width: 100%;
-                border-collapse: collapse;
-                text-align: center;
-            }
+<html lang="en">
+<head>
+    <title>Transactions</title>
+    <style>
+        table {
+            width: 100%;
+            border-collapse: collapse;
+            text-align: center;
+        }
 
-            table tr th, table tr td {
-                padding: 5px;
-                border: 1px #eee solid;
-            }
+        table tr th, table tr td {
+            padding: 5px;
+            border: 1px #eee solid;
+        }
 
-            tfoot tr th, tfoot tr td {
-                font-size: 20px;
-            }
+        tfoot tr th, tfoot tr td {
+            font-size: 20px;
+        }
 
-            tfoot tr th {
-                text-align: right;
-            }
-        </style>
-    </head>
-    <body>
-        <table>
-            <thead>
-                <tr>
-                    <th>Date</th>
-                    <th>Check #</th>
-                    <th>Description</th>
-                    <th>Amount</th>
-                </tr>
-            </thead>
-            <tbody>
-                <!-- TODO -->
-            </tbody>
-            <tfoot>
-                <tr>
-                    <th colspan="3">Total Income:</th>
-                    <td><!-- TODO --></td>
-                </tr>
-                <tr>
-                    <th colspan="3">Total Expense:</th>
-                    <td><!-- TODO --></td>
-                </tr>
-                <tr>
-                    <th colspan="3">Net Total:</th>
-                    <td><!-- TODO --></td>
-                </tr>
-            </tfoot>
-        </table>
-    </body>
+        tfoot tr th {
+            text-align: right;
+        }
+    </style>
+</head>
+<body>
+<table>
+    <thead>
+    <tr>
+        <th>Date</th>
+        <th>Check #</th>
+        <th>Description</th>
+        <th>Amount</th>
+    </tr>
+    </thead>
+    <tbody>
+    <?php foreach ($transactions as $key => $transaction): ?>
+        <tr>
+            <td><?= $transaction['date'] ?></td>
```

Comment:
same here, these need to be escaped. As I mentioned don't worry about it because it wasn't covered in the series yet, but it will be covered in the third section so you can come back & fix it once we cover it